glowing_waffles/notebooks/observe class.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>Can you please change the class name from `observe` to `Observe` or maybe `ObserveACP` or maybe `ObserveObjectACP`? Key thing is to capitalize the class name. I'm a little concerned about just using `Observe` because of the potential to collide with astroplan.</p><p><br></p><p>Also, please change method names from CamelCase to snake_case, i.e. underscores to separate words and no capitalization.</p>

 _Reply via <a href='https://app.reviewnb.com/glowing-waffle/glowing-waffles/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>